app/models/incident_report.py

```diff
@@ -61,6 +66,7 @@ def flip_coin():
                 date=fake.date_time_between(start_date="-1y", end_date="now"),
                 duration=timedelta(minutes=randint(1, 30)),
                 agency=choice(agencies),
+                user=choice(users),
```

Comment:
Hmm, I could have sworn I did this already. Is this branch up to date with master?